oceanmesh/grid.py

```diff
@@ -160,7 +160,7 @@ def create_vectors(self):
         """
         x = self.x0y0[0] + np.arange(0, self.nx) * self.dx  # ascending monotonically
         y = self.x0y0[1] + np.arange(0, self.ny) * abs(self.dy)
-        y = y[::-1]  # descending monotonically
+        # y = y[::-1]  # descending monotonically
```

Comment:
wait @krober10nd before merging. This induced a bug in the wavelength size function (now upside down). I need to rectify